### PR TITLE
AFL++ for image-rs

### PR DESCRIPTION
**Pull request by anfedotoff**

### CODE REVIEW

File: projects/image-rs/hdr-afl++.toml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 ISP RAS
```
Comment (apach301) [inline projects/image-rs/hdr-afl++.toml:1]: 2023

File: projects/image-rs/ico-afl++.toml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Copyright 2021 ISP RAS
```
Comment (apach301) [inline projects/image-rs/ico-afl++.toml:1]: 2023

**Review by apach301 — APPROVED** [review]

File: projects/image-rs/exr-afl++.toml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+target = "/sydr_script_exr @@"
+
+[aflplusplus]
+target = "/fuzz_exr"
+args = "-t 2000+ -i /corpus_exr"
```
Comment (kobrineli) [inline projects/image-rs/exr-afl++.toml:22]: do we need the `+` in timeout option here?

File: projects/image-rs/hdr-afl++.toml (hunk 1 above)
Comment (kobrineli) [inline projects/image-rs/hdr-afl++.toml:1]: 2021 -> 2023

File: projects/image-rs/ico-afl++.toml (hunk 2 above)
Comment (kobrineli) [inline projects/image-rs/ico-afl++.toml:1]: 2021 -> 2023

File: projects/image-rs/guess-afl++.toml [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+# limitations under the License.
+#
+################################################################################
+
+[sydr]
+target = "/sydr_script_guess @@"
+
+[aflplusplus]
+target = "/fuzz_guess"
+args = "-i /corpus_bmp"
```
Comment (kobrineli) [inline projects/image-rs/guess-afl++.toml:22]: maybe then specify the same corpus for the `guess.toml`?

**Review by kobrineli — APPROVED** [review]

**Pull request merged**